docs/workspaces.md

```diff
@@ -60,6 +60,12 @@ they have independent behaviour and no cluster workspace sees objects from other
 cluster workspaces. In contrast to namespace in Kubernetes, this includes non-namespaced
 objects, e.g. like CRDs where each workspace can have its own set of CRDs installed.
 
+The following cluster workspace types are created internally to support User Home Workspaces:
+
+* `Home`: the ClusterWorkspace of home workspaces - can contain any type of workspaces as children (especially universal workspaces)
+* `Homebucket`: the type of workspaces that will contains a subset of all home workspaces. - Can contain either Homebucket (multi-level bucketing) or Home workspaces
+* `Homeroot`: the workspace that will contain all the Home workspaces, spread accross buckets. - Can contain only Homebucket workspaces
```

Comment:
I would reorder with Homeroot first, then Homebucket, then Home. WDYT?

Reply:
I'm not against it.

Reply:
Do we want to include this in a dedicated section about user Home workspaces, very explaining the concept and related virtual existence and automatic creation ?